CodeStarterCamp_Week3/main.swift

```diff
@@ -8,57 +8,9 @@
 
 import Foundation
 
-struct Person {
-    var haveMoney: Int
-    var nickName: String
-    
-    //커피 주문하는 함수
-    mutating func orderCoffee(coffeePrice: Int, coffeeName: String) {
-        if self.haveMoney >= coffeePrice {
-            print("\(coffeeName) 주세요.")
-            self.haveMoney -= coffeePrice
-        }
-        else {
-            print("\(coffeePrice - self.haveMoney)원 만큼 돈이 부족하네")
-        }
-    }
-    
-}
+var misskim = Person(haveMoney: 5000, nickName: "misskim")
+var idinaloq = Person(haveMoney: 6000, nickName: "idinaloq")
+var yagombucks = CoffeeShop(revenue: 0, menu: [.espresso: 3000, .americano: 3500, .cappuccino: 4500, .cafeLatte: 5500])
 
-struct CoffeeShop {
-    var revenue: Int = 0
-    var menu: [Coffee: Int] = [:]
-    var pickUpTable: [Coffee] = []
-    var barista: Person?
-    
-    //커피 주문받고 만드는 함수
-    mutating func makeCoffee(coffeeName: Coffee) {
-        
-        if let coffeePrice = menu[coffeeName] {
-            self.revenue += coffeePrice
-            putInPickUpTable(coffeeName: coffeeName)
-        }
-        else {
-            print("해당 커피가 없습니다")
-        }
-    }
-    
-    //커피 pickUpTable에 올려놓는 함수
-    mutating func putInPickUpTable(coffeeName: Coffee) {
-        print("주문하신 \(coffeeName)가 나왔습니다.")
-        self.pickUpTable.append(coffeeName)
-    }
-    
-}
-
-enum Coffee: String{
-    case espresso = "에스프레소"
-    case americano = "아메리카노"
-    case cafeLatte = "카페라떼"
-    case cappuccino = "카푸치노"
-}
-
-var misterLee = Person(haveMoney: 3500, nickName: "lee")
-var missKim = Person(haveMoney: 10000, nickName: "kim")
-var yagombucks = CoffeeShop(revenue: 0, menu: [.espresso: 3500, .cappuccino: 4000, .cafeLatte: 4500, .americano: 5000])
-yagombucks.barista = misterLee
+misskim.order(.espresso, of: yagombucks, by: "misskim")
+idinaloq.order(.americano, of: yagombucks, by: "idinaloq")
```

Comment:
```suggestion
misskim.order(.espresso, of: yagombucks, by: misskim.nickName)
idinaloq.order(.americano, of: yagombucks, by: idinaloq.nickName)
```

매직 리터럴을 통해 값을 넣어주는 것이 아니라 만들어둔 프로퍼티를 사용하면 오타나 실수를 배제할 수 있겠습니다 😃